lib/ukboot/shutdown_req.c

```diff
@@ -54,3 +55,17 @@ int uk_boot_shutdown_req_isr(enum ukplat_gstate target)
 #endif /* __INTERRUPTSAFE__ */
 	return 0;
 }
+
+#if __INTERRUPTSAFE__ && CONFIG_LIBUKBOOT_SHUTDOWNREQ_HANDLER
+static int shutdown_req_handler(void *data)
+{
+	enum ukplat_gstate request = (enum ukplat_gstate)data;
+	int rc;
+
+	rc = uk_boot_shutdown_req_isr(request);
+	if (unlikely(rc < 0))
+		return UK_EVENT_NOT_HANDLED;
+	return UK_EVENT_HANDLED_CONT;
+}
+UK_EVENT_HANDLER(UKPLAT_SHUTDOWN_EVENT, shutdown_req_handler);
```

Comment:
`checkpatch.pl --strict` throws
```
CHECK: Please use a blank line after function/struct/union/enum declarations
#62: FILE: lib/ukboot/shutdown_req.c:70:
+}
+UK_EVENT_HANDLER(UKPLAT_SHUTDOWN_EVENT, shutdown_req_handler);
```

Reply:
This is a registration call and not a (function prototype) declaration. I think checkpatch is too strict here.